README.md

```diff
@@ -1,31 +1,28 @@
-Allen Cell react-redux app setup
-=================
+# Project name
 
-#### Install:
-```
-$ git clone git@github.com:allen-cell-animated/base-react-redux-app.git
+One line description
 
-$ ./gradlew npmInstall
-```
+---
 
-#### Configuration:
+## Description
 
-| Env var | Default | Options |
-| ------- |-------- |---------|
-|`PORT`   | 9002    |         |
-|`DEPLOYMENT_ENV`    | dev     | "dev", "staging", "production" |
+Main features in a brief descriptive text.
 
+## Installation
 
-Differences in builds by environment:
+Describe how to obtain the software and get it ready to run
 
-| Target | Sources Maps |  NODE_ENV === 'production' |
-| ------ | ------------ |  ------------------------- |
-| dev    | true         |  false                     |
-| staging| true         |  true                      |
-|production| false      |  true                      |
```

Comment:
is the above info no longer true for this project?

Reply:
I moved it into the CONTRIBUTING doc